packages/contracts/docs/core/04-contracts-overview.md

```diff
@@ -1,115 +0,0 @@
----
-title: Contracts Overview
----
-
-## A Categorization of the Core Contracts
```

Comment:
Don't we have a counterpart of this now?

Reply:
Instead of doing this classification in one page, I will do it separately
- for the core
- for the framework
in the next sub-task